### PR TITLE
Bugfix/demtests

### PR DESCRIPTION
**Pull request by jmpearl**
# Summary

- Sean caught some errors in the DEM test scripts not included in the ats. This PR updates those scripts.
- It does the following:
  - updates 3 DEM test scripts which were broken
  - Removes one duplicate DEM test script that was defunct
  - fixes the distance method of the RectangularPlanarSolidBoundary

------
### ToDo :

- [x] Annotate ``RELEASE_NOTES.md`` with notable changes.
- [x] Create LLNLSpheral PR pointing at this branch. (PR#)
- [x] LLNLSpheral PR has passed all tests.

### CODE REVIEW

File: src/DEM/SolidBoundary/RectangularPlaneSolidBoundary.cc [hunk 1]
```diff
@@ -37,9 +37,9 @@ distance(const Vector& position) const {
   const double signX = (q.x() > 0.0 ? 1. : -1.);
```
Comment (jmikeowen) [inline src/DEM/SolidBoundary/RectangularPlaneSolidBoundary.cc:37]: Minor quibble, but these recreate the sgn() functions we provide in Utilities/SpheralFunctions.hh (I know, a very generic/non-descriptive file name).

**Comment by jmikeowen** [comment]
Overall these changes look good.  My only request would be to replace the locally defined sign functions with the generic ones we provide in the Utilities directory.

File: src/DEM/SolidBoundary/RectangularPlaneSolidBoundary.cc (hunk 1 above)
Comment (jmpearl) [reply src/DEM/SolidBoundary/RectangularPlaneSolidBoundary.cc:37]: ah gotcha yeah no point in reinventing the wheel, I'll swap those out.

Comment (jmpearl) [reply src/DEM/SolidBoundary/RectangularPlaneSolidBoundary.cc]: On second look, I realized I was just implementing bounds on that Vector's values in a confusing way

Comment (jmikeowen) [reply src/DEM/SolidBoundary/RectangularPlaneSolidBoundary.cc]: Even better, looks good to me.

**Review by jmikeowen — APPROVED** [review]
Good to go

**Pull request merged**